web/rest/platform/config/api/raw/provider.yml

```diff
@@ -43,7 +43,8 @@ Ivoz\Provider\Domain\Model\BillableCall\BillableCall:
 
 Ivoz\Provider\Domain\Model\Ddi\Ddi:
   itemOperations: []
-  collectionOperations: []
+  collectionOperations: 
```

Comment:
why do we need to expose ddis at platform level?

Reply:
Because in the external calls list, there is a fkresolver pointing to Ddi's of API platform, and when i tried to get Ddi's in the initial request where the list is being loaded failed because of two reasons, the Ddi entity path was with a typo and the endpoint wasn't published.

Reply:
But it's not shown anywhere so, just skip that call